docs/configuration/settings.rst

```diff
@@ -91,7 +91,7 @@ Backends
 Kinto relies on three types of backends: storage, cache and permission. The
 settings names have a different prefix for each.
 
-For each of them, the supported services are currently PostgreSQL, Redis, and Memory.
+For each of them, the supported services are currently PostgreSQL, Redis, Memcached and Memory.
```

Comment:
This isn't true. This sentence should stay the same, along with `Memcached is also available as a cache backend.`